numpy/lib/mixins.py

```diff
@@ -179,3 +211,45 @@ def __repr__(self):
     __pos__ = _unary_method(um.positive, 'pos')
     __abs__ = _unary_method(um.absolute, 'abs')
     __invert__ = _unary_method(um.invert, 'invert')
+
+
+class NDArrayReductionsMixin(object):
+    """
+    Mixin defining all array reduction methods using __array_ufunc__.
+
+    This class implements methods for the reductions supported by ``ndarray``,
+    including ``sum``, ``min``, ``any``, etc.
+
+    It is useful for writing classes that do not inherit from `numpy.ndarray`,
+    but that should support reductions like arrays as described in :ref:`A
+    Mechanism for Overriding Ufuncs <neps.ufunc-overrides>`.
```

Comment:
This isn't really accurate - `np.sum()` will work just fine on any object defining `__array_ufunc__` - all this is doing is adding alias methods.

Reply:
True, I'll change that.